lib/bundler/index.js

```diff
@@ -0,0 +1,132 @@
+// Avoid Browserifying these dependencies
+let crypto, fs, path, through;
```

Comment:
Let's stick to `var` for now here and elsewhere in this file, for consistency with the rest of the Derby code, until we get around to converting it to ES6+ syntax.